### DISCUSSION TITLE
Any way to make remix treat new file extension as an asset?

### DISCUSSION THREAD

**Discussion by bschlenk** [remix-run/remix#4140] [Q&A]
Hi! I'm trying to import a `.glb` model into my app, but remix is throwing "ERROR: no loader is configured for .glb files". Vite lets you extend the supported extensions with [assetsInclude](https://vitejs.dev/config/shared-options.html#assetsinclude). Does remix have something similar?

**Comment by sergiodxa** [comment]
There's no way right now, check the discussion of this PR https://github.com/remix-run/remix/pull/4094 where some ideas where suggested.
